core/dbt/contracts/graph/nodes.py

```diff
@@ -1399,6 +1411,28 @@ class Group(BaseNode):
     resource_type: NodeType = field(metadata={"restrict": [NodeType.Group]})
 
 
+# ====================================
+# SemanticModel and related classes
+# ====================================
+
+
+@dataclass
+class NodeRelation(dbtClassMixin, Replaceable):
```

Comment:
This a lot like the StateRelation class and a number of other relation classes that we have. I personally don't like using "alias" in these classes instead of "identifier" because without an actual identifier there's nothing for it to be an alias of. Can we use "identifier" and create an "alias" property as a synonym?

Reply:
In this case, a name change would affect the serialization format we're targeting, unless we control it another way. Is there still flexibility on that format, Quigley? What do you think about Gerda's suggestion?

Reply:
fwiw - I also prefer `identifier` to `alias` here

Reply:
I agree that we should make the move to `identifier`. Is it okay if we do that as a follow up task though? It'd require the change in DSI, re-releasing DSI, and then making the update here. Not the biggest deal. There shouldn't be any gotchas. However, as this is an under the hood change and we're on a tight deadline, I think we could push this to extra work to be done after Wednesday (June 7th)